CoC.txt

```diff
@@ -54,6 +54,76 @@ Members of the community should not hesitate to contact the Community Council if
 they feel someone has violated this Code of Conduct, or if they have questions
 or concerns.
 
+================================================================================
+Behavioral Guidelines
+================================================================================
+
+As a member of the community, we ask you to:
+
+- Be friendly and patient. Your work will be used by other people, and you in
+  turn will depend on the work of others. Any decision you take will affect
+  human beings, and you should take those consequences into account when making
+  decisions. Remember that our community is international, and that the people
+  you interact with may be communicating with you in a language that is not
+  their native tongue.
+
+- Assume good faith and good intentions. Disagreements — both social and
+  technical — happen all the time. Our diverse backgrounds give us the ability
+  to see things from many different perspectives, but they can also lead to
+  misunderstandings. It is important that we resolve disagreements
+  constructively, and that we do not jump to conclusions about each other's
+  motives.
+
+- Be welcoming, considerate, and respectful. We strive to be a community that
+  welcomes and supports people of all backgrounds and identities. This includes,
```

Comment:
What about **People of all backgrounds and identities deserve to be welcomed and supported**